Feat: Adds xypad and an example for it

bboettcher3 opened the pull request:
* Adds XY pad view
    * Supports disabling
    * Supports light/dark themes
* Adds an example for the new view
<img width="269" alt="Screenshot 2024-04-07 at 1 41 44 PM" src="https://github.com/vizia/vizia/assets/13266420/8cd76a11-0320-419a-a87a-6042b9da333d">
<img width="269" alt="Screenshot 2024-04-07 at 11 41 26 AM" src="https://github.com/vizia/vizia/assets/13266420/4ced213b-998f-4386-aff7-506b2143e1c8">

bboettcher3 commented:
One odd behavior is that the xypad element and thumb CSS classes don't change the border properties I want it to on dark theme.. Maybe someone could check if I'm missing something there? Other properties like background-color change correctly, but the border-color has no effect.

geom3trik commented:
> One odd behavior is that the xypad element and thumb CSS classes don't change the border properties I want it to on dark theme.. Maybe someone could check if I'm missing something there? Other properties like background-color change correctly, but the border-color has no effect.

It looks like you've set a border colour for it in rust code. This will override any css styling on the element.

bboettcher3 commented:
> It looks like you've set a border colour for it in rust code. This will override any css styling on the element.

Ahh, fixed that and updated the image above with the new theme, thanks!

magnetophon commented:
I totally missed this merge!
Thanks, @bboettcher3 and @geom3trik!
